Code/GraphMol/MolStandardize/Wrap/testMolStandardize.py

```diff
@@ -1780,6 +1780,38 @@ def test26PipelineAllowEmptyMoleculesOption(self):
     self.assertEqual(result.stage, rdMolStandardize.PipelineStage.COMPLETED)
     self.assertEqual(result.status, rdMolStandardize.PipelineStatus.NO_EVENT)
 
-
+  def testCustomScoreFuncs(self):
```

Comment:
I think it wouldn't be a terrible idea to add a test/example which shows using the custom scoring substructs together with a python scoring function passed to `Canonicalize()`

Reply:
Will do.